requirements.txt

```diff
@@ -1,3 +1,3 @@
-mkdocs
-mkdocs-material
-mkdocs-minify-plugin
\ No newline at end of file
+mkdocs>=1.0,<2.0
+mkdocs-material>=9.0,<10.0
+mkdocs-minify-plugin<1.0
```

Comment:
why the limitations? What happens in 2.0?

Reply:
Most likely breaking changes in the next major version as they seem to go with semantic versioning. Just want to avoid that